src/web/local.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+volumes:
+  local_postgres_data:
+
+services:
+
+  db:
+    image: postgres
+    restart: always
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+
+  app:
+    build: .
```

Comment:
This can work, but let be inspired by this:

```
services:

  db:
    image: "postgres:alpine"
    restart: always
    environment:
      POSTGRES_USER: postgres
      POSTGRES_PASSWORD: postgres
    networks:
      postgres:

  app:
    build: .
    ports:
      - "8080:8080"
    image: teap
    depends_on:
      - db
    networks:
      postgres:

networks:
  postgres:
```

Reply:
done, also added separate env variables for postgres user, password